operator/internal/controller/redpanda/role_controller_test.go

```diff
@@ -59,9 +59,10 @@ func TestRoleReconcile(t *testing.T) { // nolint:funlen // These tests have clea
 	}
 
 	for name, tt := range map[string]struct {
-		mutate            func(role *redpandav1alpha2.RedpandaRole)
-		expectedCondition metav1.Condition
-		onlyCheckDeletion bool
+		mutate                   func(role *redpandav1alpha2.RedpandaRole)
+		expectedCondition        metav1.Condition
+		onlyCheckDeletion        bool
+		expectedStatusPrincipals []string
```

Comment:
Am I missing something or `expectedStatusPrincipals` is not used in the test case definition?